modules.json

```diff
@@ -23,8 +23,7 @@
                     "csvtk/join": {
                         "branch": "master",
                         "git_sha": "666652151335353eef2fcd58880bcef5bc2928e1",
-                        "installed_by": ["modules"],
-                        "patch": "modules/nf-core/csvtk/join/csvtk-join.diff"
```

Comment:
I'm not sure if removing all patch files was intended or an artifact of the template update, leaving this comment to make sure that this is not a mistake. It also happens with other modules 👀

Reply:
UHHH massive catch!! Thanks :D